frontend/src/app/game/components/graphs/graphs.component.ts

```diff
@@ -137,9 +137,9 @@ export class GraphsComponent implements AfterViewInit {
         {
           label: new Date(gs.date),
           value: gs.stats.estimatedResistant.total,
-          tooltipLabel: (value: number) => `Imunní po nemoci: ${formatNumber(value)}`,
+          tooltipLabel: (value: number) => `Krátce po nemoci: ${formatNumber(value)}`,
```

Comment:
We should not be changing this, right?

Reply:
I think I already removed this patch from the PR. But let me double check

Reply:
Done